provers/debug/prover.go

```diff
@@ -0,0 +1,157 @@
+package debug
+
+import (
+	"context"
+	fmt "fmt"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
+	"github.com/cosmos/ibc-go/v8/modules/core/exported"
+
+	//mocktypes "github.com/datachainlab/ibc-mock-client/modules/light-clients/xx-mock/types"
+	"github.com/hyperledger-labs/yui-relayer/core"
+	"github.com/hyperledger-labs/yui-relayer/log"
+)
+
+func debugFakeLost(ctx context.Context, chain core.Chain, queryHeight exported.Height) error {
+	if val, ok := os.LookupEnv("DEBUG_RELAYER_MISSING_TRIE_NODE_HEIGHT_PROVER"); ok {
+		s := strings.Split(val, " ")
+		if len(s) != 2 {
+			fmt.Printf("malformed DEBUG_RELAYER_MISSING_TRIE_NODE_HEIGHT_PROVER: Expected format: <chainid> <height threshold>\n")
+			return nil
+		}
```

Comment:
When the `DEBUG_RELAYER_MISSING_TRIE_NODE_HEIGHT` variable is set but its value is wrong, this function should return an error here.